tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1937,15 +1937,9 @@ def update_llm_args_with_extra_dict(
         "quant_config": QuantConfig,
         "calib_config": CalibConfig,
         "build_config": BuildConfig,
-        "kv_cache_config": KvCacheConfig,
```

Comment:
To make sure I'm clear, this is because these can be feed to the underlying pydantic `BaseModel` directly without any special handing right?

Reply:
Yes, you are right. The remaining ones are not Pydantic, thus we leave them there to do explicit parsing.